### PR TITLE
Refactor laminas

### LINKED ISSUE #55: Zend migration to Laminas
The Zendframework has migrated to Laminas.
Are there any plans to replace the zend dependencies with laminas?

### PR DESCRIPTION
**Pull request by nkop**
This PR fixes #55 by replacing the zend-framework packages with their laminas counterparts

Changed:
- Replace the zend-framework packages with laminas
- Update copyright year to fix PHPCS errors/warnings

### CODE REVIEW

**Comment by nkop** [comment]
Scrutinizer seems to crash because it runs out of memory.

**Comment by imonteiro** [comment]
On composer.json, should change `zf` to `laminas`.

File: composer.json [hunk 1]
```diff
@@ -17,10 +17,10 @@
     ],
     "require": {
         "php": "^7.2",
+        "laminas/laminas-cache": "^2.7",
+        "laminas/laminas-stdlib": "^2.2 || ^3.0",
         "league/flysystem": "^1.0.41",
```
Comment (imonteiro) [inline composer.json]: @basz should we change to latest release (1.0.64)?

Comment (basz) [reply composer.json]: any particular reason why we should? I don't see why...

**Comment by basz** [comment]
LGTM! Thanks!

About to merge today, waiting for a response from @imonteiro

File: composer.json (hunk 1 above)
Comment (imonteiro) [reply composer.json]: I'm not composer specialist, but I'm wondering if it's possible to require the minimum version (1.0.41) but if nothing is specificed in the project the composer will install the latest stable version in 1.0.* (at the moment, 1.0.64).

Make sense?

Comment (basz) [reply composer.json]: The caret (^) mean any from 1.0.41 and below 2.0.0. So we are good.

**Pull request merged**